tests/test_util.py

```diff
@@ -323,3 +323,22 @@ def test_unfulfilled_kwargs(self):
         exc = pytest.raises(ValueError, check_callable_args, func, [1], {})
         assert str(exc.value) == ('The following keyword-only arguments have not been supplied in '
                                   'kwargs: y')
+
+
+def not_a_coro(x):
```

Comment:
These should probably go inside the new test class.

Reply:
Amended.